dcrjson/dcrwalletextcmds.go

```diff
@@ -532,6 +534,30 @@ func NewWalletInfoCmd() *WalletInfoCmd {
 	return &WalletInfoCmd{}
 }
 
+type OutputDestination struct {
+	Address       string
+	Script        []byte
+	ScriptVersion uint32
+}
+
+// SweepAccountCmd defines the sweep account JSON-RPC command.
+type SweepAccountCmd struct {
+	SourceAccount         uint32
+	RequiredConfirmations uint32
+	FeePerKb              dcrutil.Amount
+	ChangeDestination     *OutputDestination
```

Comment:
Is ChangeDestination where the account is swept to?  I'm assuming so, but this needs a different name.  Perhaps just `DestinationAddress string` (see comments above about not needing the scripts).